pardata/loaders/_json.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2021 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"JSON file loaders."
+
+
+from typing import cast, Dict, Union
+import json
+
+from .. import typing as typing_
+from ..schema import SchemaDict
+from ._base import Loader
+
+
+class JSONLoader(Loader):
+    """JSON file to :class: list object loader."""
+
+    def load(self, path: Union[typing_.PathLike, Dict[str, str]], options: SchemaDict) -> list:
+        """
+        :param path: The path to the JSON file.
+        :param options: None for JSON loader.
+        :raises TypeError: ``path`` is not a path object.
+        :return: Data loaded into a :class: list object.
```

Comment:
Are you sure it'll be always a list?

Reply:
I wrap it with a list, otherwise it is possible to return different types

Reply:
It's a bit strange to force wrapping the returned value of `json.load` with a list. How about just returning `json.load`? The return type of `json.load` is `Any`: https://github.com/python/typeshed/blob/aa92ca91383fad6d85e2d9926014cd6b66fb53ee/stdlib/json/__init__.pyi#L57